src/app/lib/utilities/ichefURL/index.js

```diff
@@ -22,7 +23,7 @@ const buildPlaceholderSrc = (src, resolution) => {
   return `https://${newUrl.join('/')}`;
 };
 
-const buildIChefURL = ({ originCode, locator, resolution, isWebP = false }) => {
+const buildIChefURL = ({ originCode, locator, resolution }) => {
```

Comment:
SInce we've removed the `isWebP` param from this function, we need to make sure that it is also removed anywhere we call it from

Reply:
removed